### PR TITLE
feat: The Cell Count chart data is abnormal

### PR DESCRIPTION
**Pull request by zhb666**
### issues：https://github.com/Magickbase/ckb-explorer-public-issues/issues/318
The result obtained by adding live cells and dead cells together is not accurate when displayed on the chart as "all cells". Therefore, we should update the chart to change it to have three labels (live, dead, all).

image

### CODE REVIEW

File: src/locales/zh.json [hunk 1]
```diff
@@ -88,6 +88,7 @@
       "new_dao_withdraw": "每日新增解锁",
       "circulation_ratio": "锁定量占当前流通量比率",
       "live_cell": "未花费 Cell",
+      "dead_cell": "已死亡 Cell",
```
Comment (Keith-CY) [inline src/locales/zh.json:91]: This word is inelegant in Chinese, how about
```suggestion
      "dead_cell": "已消费 Cell",
```

> Each Cell can be consumed only once. A non-consumed cell is known as a live cell. A consumed cell is known as a dead cell. Once dead, the cell can no longer be used.

Ref: https://docs.nervos.org/docs/basics/concepts/cell-model/

**Review by Keith-CY — APPROVED** [review]

**Review by IronLu233 — APPROVED** [review]

**Comment by Keith-CY** [comment]
Please have a review @FrederLu @silySuper @littleLip520

**Comment by FrederLu** [comment]
image

After seeing the `Cell Count` table, I have 2 questions:
1. The labeling order is inconsistent with the order in the icon. Is it more intuitive and easy to understand if using the same order?
2. Is `Y-axis` only for `All Cells`? When the current nodes of `Live cell` and `Dead cell` are mapped to `Y-axis` through screenshots, it is found that the data displayed are inconsistent.

`Cell Count` needs to indicate whether it should be the respective occupancy of `Live cell` and `Dead cell` in `All cells` at the current node.

**Comment by zhb666** [comment]
> image
> After seeing the `Cell Count` table, I have 2 questions:
> 
> 1. The labeling order is inconsistent with the order in the icon. Is it more intuitive and easy to understand if using the same order?
> 2. Is `Y-axis` only for `All Cells`? When the current nodes of `Live cell` and `Dead cell` are mapped to `Y-axis` through screenshots, it is found that the data displayed are inconsistent.
> 
> `Cell Count` needs to indicate whether it should be the respective occupancy of `Live cell` and `Dead cell` in `All cells` at the current node.



> image
> After seeing the `Cell Count` table, I have 2 questions:
> 
> 1. The labeling order is inconsistent with the order in the icon. Is it more intuitive and easy to understand if using the same order?
> 2. Is `Y-axis` only for `All Cells`? When the current nodes of `Live cell` and `Dead cell` are mapped to `Y-axis` through screenshots, it is found that the data displayed are inconsistent.
> 
> `Cell Count` needs to indicate whether it should be the respective occupancy of `Live cell` and `Dead cell` in `All cells` at the current node.

We greatly appreciate the first suggestion. As for the second suggestion, since it directly displays the data, we need to communicate and confirm whether there is an issue with the data.

**Comment by zhb666** [comment]
> image
> After seeing the `Cell Count` table, I have 2 questions:
> 
> 1. The labeling order is inconsistent with the order in the icon. Is it more intuitive and easy to understand if using the same order?
> 2. Is `Y-axis` only for `All Cells`? When the current nodes of `Live cell` and `Dead cell` are mapped to `Y-axis` through screenshots, it is found that the data displayed are inconsistent.
> 
> `Cell Count` needs to indicate whether it should be the respective occupancy of `Live cell` and `Dead cell` in `All cells` at the current node.

Regarding the first suggestion, I successfully made the requested modifications, but I realized it was incorrect. This value cannot be set because the candlestick chart changes with the height, and it should be identified based on color instead.

**Comment by FrederLu** [comment]
image
image

Comparing the data of `2022/4/28/` and `2022/5/1`, it is found that the amount of `All Cells` and `Dead cell` has increased, but the amount of `Live cells` has decreased, but from the figure, `Live cell` cell` is also rising, it feels wrong.

**Comment by silySuper** [comment]
1.A cell contains two status：dead or live ，total cell=dead cell+live cell ，why some days shows total cell >dead cell+live cell?

截屏2023-06-16 10 36 39

2.In the chart,I can not find 2022-05-25 data,what is the reason?


https://github.com/Magickbase/ckb-explorer-frontend/assets/25971273/c1901d96-c241-42e4-9072-e3dd22db23b0

**Comment by zhb666** [comment]
> image image
> Comparing the data of `2022/4/28/` and `2022/5/1`, it is found that the amount of `All Cells` and `Dead cell` has increased, but the amount of `Live cell` has decreased, but from the figure, `Live cell` cell` is also rising, it feels wrong.

It needs to be revalidated. This time, the issue with the scale of the candlestick chart has been fixed.

**Comment by zhb666** [comment]
> 1.A cell contains two status：dead or live ，total cell=dead cell+live cell ，why some days shows total cell >dead cell+live cell?
> 
> 截屏2023-06-16 10 36 39
> 2.In the chart,I can not find 2022-05-25 data,what is the reason?
> 
>  2023-06-16.10.59.41.mov

 If there is data available for that date, it will be displayed. There is data available for May 25, 2022, in the mainnet test.
image

**Comment by FrederLu** [comment]
> > image image
> > Comparing the data of `2022/4/28/` and `2022/5/1`, it is found that the amount of `All Cells` and `Dead cell` has increased, but the amount of `Live cell` has decreased, but from the figure, `Live cell` cell` is also rising, it feels wrong.
> 
> It needs to be revalidated. This time, the issue with the scale of the candlestick chart has been fixed.

The above problems have been improved. But now this graph does not feel like a stacked graph (`live cell + dead cell = all cells`), it is more like an overlapping graph.
image

refer to  https://explorer.nervos.org/charts/liquidity

**Comment by zhb666** [comment]
> > > image image
> > > Comparing the data of `2022/4/28/` and `2022/5/1`, it is found that the amount of `All Cells` and `Dead cell` has increased, but the amount of `Live cell` has decreased, but from the figure, `Live cell` cell` is also rising, it feels wrong.
> > 
> > 
> > It needs to be revalidated. This time, the issue with the scale of the candlestick chart has been fixed.
> 
> The above problems have been improved. But now this graph does not feel like a stacked graph (`live cell + dead cell = all cells`), it is more like an overlapping graph. image
> 
> refer to https://explorer.nervos.org/charts/liquidity

 It seems that the issue is related to the data. Let's continue the discussion on this matter in the following link：https://github.com/Magickbase/ckb-explorer-public-issues/issues/347

**Comment by zhb666** [comment]
image
This is the updated version after the changes. It needs to be tested. Thank you.
@FrederLu

**Comment by FrederLu** [comment]
> image This is the updated version after the changes. It needs to be tested. Thank you. @FrederLu

Now it looks like a stacked graph.
It can be seen from the screenshot that there are `live cell` and `dead cell` information, but the `All Cells` information cannot be seen in the unit time of the chart.
Although we know that the sum of living cells and dead cells is all the cells, it is better to use the corresponding icon to display it to make the information of all cells more eye-catching.

image

**Comment by zhb666** [comment]
> > image This is the updated version after the changes. It needs to be tested. Thank you. @FrederLu
> 
> Now it looks like a stacked graph. It can be seen from the screenshot that there are `live cell` and `dead cell` information, but the `All Cells` information cannot be seen in the unit time of the chart. Is it possible to consider using a line with the color of all cells on the top of the dead cell area? In this way, the corresponding icon information is available.
> 
> image

image
It has been bolded.

**Review by FrederLu — APPROVED** [review]

**Pull request merged**